numpy/core/tests/test_multiarray.py

```diff
@@ -4101,19 +4101,6 @@ class TestArgmax:
         ([0, 1, 2, np.nan, 3], 3),
         ([np.nan, 0, 1, 2, 3], 0),
         ([np.nan, 0, np.nan, 2, 3], 0),
-        ([0, 1, 2, 3, complex(0, np.nan)], 4),
```

Comment:
The removed tests should be put into `test_deprecations.py` I believe.

Reply:
Ah I see will look into that